plugins/Dash/ScopeStyle.qml

```diff
@@ -26,17 +28,22 @@ import Ubuntu.Components 1.3
  */
 
 QtObject {
+
+    property var gsettings: GSettings {
+          schema.id: "com.ubuntu.touch.system-settings"
+      }
+
     /// Style object passed from the scope
     property var style: Object()
 
     /// Color used for text and symbolic icons
-    readonly property color foreground: "foreground-color" in style ? style["foreground-color"] : d.defaultDark
+    property color foreground: gsettings.dashBackground ? "white" : d.defaultDark
```

Comment:
why was "readonly" removed?
Then, with change, scopes are no longer able to chose their preferred background colour.
Maybe you wanted to write:
```
readonly property color foreground: gsettings.dashBackground ? "white" : ("foreground-color" in style ? style["foreground-color"] : d.defaultDark)
```
Though I wonder, why force white color? What if the background image is mostly white?